styles/components/JobDetails.module.scss

```diff
@@ -0,0 +1,21 @@
+@import '../globals.scss';
+
+.JobDetails {
+  .subtitle {
+    margin-top: -16px;
+    color: lighten($text-color, 30);
+    font-size: 14px;
+  }
+  .howToApply {
```

Comment:
Remember to be consistent in the naming, camel case, or everything in lowercase.

Reply:
Right! I have to get my class names in order.
Let make them all lowercase with hyphens 🚀